browser/ui/webui/settings/brave_appearance_handler.cc

```diff
@@ -99,6 +104,15 @@ void BraveAppearanceHandler::GetIsSuperReferralActive(
                             base::Value(IsSuperReferralActive(profile_)));
 }
 
+void BraveAppearanceHandler::GetIsBinanceSupported(
+    const base::ListValue* args) {
+  CHECK_EQ(args->GetSize(), 1U);
+
+  AllowJavascript();
+  ResolveJavascriptCallback(args->GetList()[0],
+      base::Value(::binance::IsBinanceSupported(profile_)));
```

Comment:
I don't believe the global scope `::` operator is needed here.

Reply:
removed